### DISCUSSION TITLE
Cannot use elasticSearch REST API from remote

### DISCUSSION THREAD

**Discussion by jonathanyaod3** [Security-Onion-Solutions/securityonion#12361] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

32Gb

### Storage for /

500G

### Storage for /nsm

500G

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I can run curl -k https://192.168.87.163:9200/_cat/indices -u userName:PASSWORD in Security Onion local host. But when I try to run same elasticSearch REST API in remote(POSTMAN), it returns  error message as attachment. 
As my version is 2.5.40, there is not so-allow command available, and I am not sure which firewall hostgroups maps to Elasticsearch REST API (port 9200), so I add IP range to searchnode, managesearch and fleet, and elasticsearch config is enabled.
Please advise what configuration I missed, many thank.

![Screenshot 2024-02-14 151155](https://github.com/Security-Onion-Solutions/securityonion/assets/79876505/0e17cb10-03df-43d6-b6e3-de1d45b4f055)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by petiepooo** [comment]
Enabling advanced settings will give you access to the "elasticsearch_rest" hostgroup.

**Comment by jonathanyaod3** [comment]
Hi Pete,

Thanks a lot for your response.

I am a newcomer to Security Onion. May I know how to enable advanced
settings?

Regards

Jonathan

**Comment by petiepooo** [comment]
SOC customization is described [here](https://docs.securityonion.net/en/2.4/soc-customization.html).  You select Administration and Configuration on the left pane to get there.  At the top of the first screenshot, just to the right of where it says "Grid Configuration," there's a dropdown labeled "Options."  Clicking that allows you to "show all configurable settings, including advanced settings."  Then scroll down to expand firewall and hostgroups and you'll see the additional groups that are available.
